test/cli.js

```diff
@@ -0,0 +1,90 @@
+import test from 'ava';
+import tempfile from 'tempfile';
+import path from 'path';
+import fs from 'fs';
+import execa from 'execa';
+import fileExists from 'file-exists';
+
+global.Promise = Promise;
+
+function read() {
+	return fs.readFileSync(path.join.apply(path, arguments), 'utf8');
+}
+
+test.beforeEach(t => {
+	t.context.tmp = tempfile();
+});
+
+test('missing file operands', async t => {
+	try {
+		await execa('../cli.js');
+	} catch (err) {
+		t.true(err.code !== 0);
+		t.true(/`src` and `dest` required/.test(err.message));
```

Comment:
Why not use `t.throws` on these? It's not really important to check the code as it will only be an error if it's not `0`.